client/dashboard/components/page-header/style.scss

```diff
@@ -0,0 +1,38 @@
+@import "@wordpress/base-styles/variables";
+
+.client-dashboard-components-page-header-heading {
+	font-size: $font-size-2x-large;
+	line-height: $font-line-height-x-large;
+}
+
+.client-dashboard-components-page-header-decoration {
+	display: inline-flex;
+	width: $grid-unit-40;
+	height: $grid-unit-40;
+	flex-shrink: 0;
+	align-items: center;
+	justify-content: center;
+
+	svg {
+		fill: $gray-700;
+		flex-shrink: 0;
+		width: $grid-unit-30;
+		height: $grid-unit-30;
+	}
+
+	img {
+		flex-shrink: 0;
+		width: 100%;
+		height: 100%;
+		object-fit: cover;
+		border-radius: $radius-small;
+	}
+}
+
+.client-dashboard-components-page-header-actions {
+	flex-shrink: 0;
+}
+
+.client-dashboard-components-page-header-description {
+	max-width: 75ch;
```

Comment:
This would ideally be a variable, but I'm not sure where it should live. 

I also got a linting error for using `ch`, but I'm not sure why. Support [seems good](https://caniuse.com/ch-unit).

Reply:
There is currently a rule for the allowed units and `ch` is not part of it. It could be that there wasn't good support when the rule was introduced, so we could revisit this. For now I added a disable line for the linter.